### DISCUSSION TITLE
Button color contrast ratio

### DISCUSSION THREAD

**Discussion by angelinetran** [indiana-university/rivet-source#744] [Feature and enhancement requests]
![Screenshot 2024-03-20 at 10 59 25 AM](https://github.com/indiana-university/rivet-source/assets/1852539/1951f4cd-2743-47b0-b9bb-7be84d24855f)

I don't know about the other colors, but the [blue primary button](https://rivet.iu.edu/components/button/) has a color contrast ratio of 6.56:1. Siteimprove suggests a color ratio of 7:1 or higher for Level AAA Conformance.

**Comment by levimcg** [comment]
Hey, @angelinetran thanks for the heads up. This has been reported several times. Currently, University policy requires that we meet AA standards. This issue is based on AAA criteria which is not at present a requirement.

**Reply by angelinetran** [reply]
ok! It's bringing down our SiteImprove a11y score, so I might....go rogue and override the colors.
